fetch_submissions.py

```diff
@@ -141,13 +145,22 @@ def save_submission_as_file(submission):
 
 
 if __name__ == '__main__':
-    if (len(sys.argv) != 2):
-        print('Usage: python3 fetch_submissions.py <username>')
+    parser = argparse.ArgumentParser(description="A simple script to download your fastest, accepted solutions from Leetcode!")
+    parser.add_argument('--retry_count', type=int, default=5, help="maximum number of incorrect logins allowed")
```

Comment:
I don't see why would a user want to pass in `retry_count` as an arg when running the script.  Like which use would go "I guess this time I need 10 retries". And anyway, if the script quits after 3 retries, its simple to just run it again. So, I would see that as a redundant addition to the script. Unless you have valid use cases that I am unable to foresee.

Reply:
When you put it that way, it doesn't make much sense to put a `retry_count` as a script argument. I misunderstood [your comment about the caller of the login function to pass the `retry_count` parameter](https://github.com/sushinoya/leetcode-downloader/pull/2#discussion_r331738249) as making it more clear for users to specify it. I'll make the change.